website/common/locales/en/settings.json

```diff
@@ -16,6 +16,7 @@
     "startAdvCollapsed": "Advanced Options in tasks start collapsed",
     "startAdvCollapsedPop": "With this option set, Advanced Options will be hidden when you first open a task for editing.",
     "dontShowAgain": "Don't show this again",
+    "suppressLoginIncentiveModal": "Don't show popup when checking in",
```

Comment:
Yes, I guess something like this:
- **Don't show "Check-In Prize" popup**
- **Don't show "Check-In Counter" popup**

They'd probably benefit from an on-hover effect to give more explanations about what each are, as we do for some of the other settings (see underlines in the screenshot below). Perhaps:

- **Each day when you use Habitica, your "Check-In" counter is increased by one. At certain numbers, you receive [prizes](http://habitica.wikia.com/wiki/Daily_Check-In_Incentives)! This setting controls whether you receive a notification each time you get a prize.**
- **Each day when you use Habitica, your "Check-In" counter is increased by one. At certain numbers, you receive [prizes](http://habitica.wikia.com/wiki/Daily_Check-In_Incentives)! This setting controls whether you receive a notification each time your counter has increased but you haven't got a prize yet.**

Both of those could probably do with improvement though.


However.... 

I'm wondering if we can reverse the way this works and make the messages say "Show .... popup" with the checkbox ticked by default. That feels more natural. Below is a screenshot showing the current settings as they are for a new account. Making this "Show..." would be inconsistent with the other items at the bottom of the list, but consistent with the "Show Header" option at the top. The inconsistency is undesirable but we could move towards doing it for the other settings too. I feel it's a nicer way of presenting it. @lemoness what do you think? The wording would then be
- **Show "Check-In Prize" popup**
- **Show "Check-In Counter" popup**
with the on-hover text being the same as above. If we put these two new settings directly under "Set Dailies default to 'due' tab" then the underlining would be consistent with the items above it, which might help to reduce the impact of the "Show / Don't show" inconsistency.

![image](https://cloud.githubusercontent.com/assets/1495809/22803331/37d5fd3a-ef60-11e6-89d3-3e964f99a255.png)